File: sdks/python/examples/market-data/search_events.py
import pmxt

def main():
    poly = pmxt.Polymarket()
    # kalshi = pmxt.Kalshi() 

    # Search for broad events (groups of markets)
    events = poly.search_events('Fed Chair')
    
    for event in events:
        print(f"Event: {event.title}")
        print(f"  Markets: {len(event.markets)}")
        
        # Example: Drill down into markets
        # for market in event.markets:
        #     print(f"    - {market.title}")

if __name__ == "__main__":
    main()

File: sdks/typescript/examples/market-data/search_events.ts
import pmxt from 'pmxtjs';

const main = async () => {
    const poly = new pmxt.Polymarket();
    // const kalshi = new pmxt.Kalshi();

    const events = await poly.searchEvents('Fed Chair');

    events.forEach(event => {
        console.log(`Event: ${event.title}`);
        console.log(`  Markets: ${event.markets.length}`);
    });
};

main();
